hookman/hookman_generator.py

```diff
@@ -279,18 +287,34 @@ def generate_plugin_package(
 
         self._validate_package_folder(artifacts_dir, assets_dir)
         self._validate_plugin_config_file(assets_dir / "plugin.yaml")
-        version = PluginInfo(assets_dir / "plugin.yaml", hooks_available=None).version
+        plugin_info = PluginInfo(assets_dir / "plugin.yaml", hooks_available=None)
 
         if sys.platform == "win32":
             shared_lib_extension = "*.dll"
-            hmplugin_path = dst_path / f"{package_name}-{version}-win64.hmplugin"
+            hmplugin_path = dst_path / f"{package_name}-{plugin_info.version}-win64.hmplugin"
         else:
             shared_lib_extension = "*.so"
-            hmplugin_path = dst_path / f"{package_name}-{version}-linux64.hmplugin"
+            hmplugin_path = dst_path / f"{package_name}-{plugin_info.version}-linux64.hmplugin"
+
+        contents = (assets_dir / 'plugin.yaml').read_text()
+        if extras_defaults is not None:
+            import strictyaml
+
+            contents_dict = strictyaml.load(contents, PLUGIN_CONFIG_SCHEMA)
+            if 'extras' not in contents_dict:
```

Comment:
I believe this is simpler:

```python
extras = extras_defaults.copy()
extras.update(contents_dict.get('extras', {}))
contents_dict['extras'] = extras_defaults
```

Reply:
Yes, better (last line is `contents_dict['extras'] = extras` though)